content/id/docs/concepts/services-networking/ingress-controllers.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Kontroler <i>Ingress</i>
+content_template: templates/concept
+weight: 40
+---
+
+{{% capture overview %}}
+
+Agar Ingress dapat bekerja sebaagaimana mestinya, 
+sebuah kluster harus memiliki sebuah kontroler Ingress.
+
+Berbeda dengan kontroler-kontroler lainnya yang dijalankan 
+sebagai bagian dari <i>binary</i> `kube-controller-manager`, kontroler Ingress  
+tidak secara otomatis dijalankan di dalam kluster. Kamu bisa menggunakan 
```

Comment:
Hi @irvifa. Just looking at the new Bahasa Indonesia docs! Great job.
When I looked at the preview, this page listed as English. I was expecting to see Indonesian for this file?
Just a thought, do you need to use HTML markup in the markdown file, `<i>` ? Could use markdown instead, *binary*? I am not sure if the English version emphasizes `binary`, or is that the way you are emphasizing the English? Also, I don't think the front matter, `title` field needs emphasis.

Reply:
@kbhawkey On it, I create two separate PR tho, I'll push it to that PR later, it's because this lacks of `_index.md` inside the `concepts` directory.
For the `<i>` tag, we use that because we couldn't find a word that can describe that better in Bahasa Indonesia, great.. never thought about using the equivalent italic in markdown cc @girikuncoro

Reply:
It works like a charm, I can use the markdown 👍 and the translations working after I add `index.md` on the concepts. Will push it this afternoon. Thanks for noticing 🙇

Reply:
Pushed #13958